ui/pages/swaps/transaction-settings/transaction-settings.js

```diff
@@ -269,12 +266,12 @@ export default function TransactionSettings({
                   title={notificationTitle}
                   titleProps={{ 'data-testid': 'swaps-banner-title' }}
                 >
-                  <Typography
-                    variant={TypographyVariant.H6}
+                  <Text
+                    variant={TextVariant.bodySm}
                     testId="mm-banner-alert-notification-text"
```

Comment:
This should fix failing e2e tests

```suggestion
                    data-testid="mm-banner-alert-notification-text"
```